frontend/__tests__/routes/_public+/date-of-birth.test.tsx

```diff
@@ -94,6 +97,8 @@ describe('_public.apply.id.date-of-birth', () => {
       vi.mocked(isPast).mockReturnValueOnce(true);
       vi.mocked(parse).mockReturnValueOnce(new Date(1959, 0, 1));
       vi.mocked(differenceInYears).mockReturnValueOnce(65).mockReturnValueOnce(65);
+      vi.mocked(parseDateString).mockReturnValue({ year: '1959', month: '01', day: '01' });
+      vi.mocked(getAgeFromDateString).mockReturnValueOnce(new Date().getFullYear() - 1959);
```

Comment:
This shouldn't return a value based on `new Date()`. It should return a consistent value across all test runs.

Reply:
True, will adjust.

Reply:
Fixed